.travis.yml

```diff
@@ -25,16 +30,22 @@ matrix:
         - CMD="./.ci/travis.sh patch-diff-report-tool"
 
     - jdk: openjdk8
+      arch: amd64
+      dist: xenial
```

Comment:
Just curious why we don't use this arch for everything since it seems to fix some issues? Is this not one of the free ones?

Reply:
No it is not free tier arch, that arch need paid credits to be used, we always need to ask for new credit. In next PR there is a fix